packages/odata-service-inquirer/src/prompts/datasources/service-url/connectionValidator.ts

```diff
@@ -0,0 +1,280 @@
+import { isAppStudio } from '@sap-ux/btp-utils';
+import { ERROR_TYPE, ErrorHandler } from '../../../error-handler/error-handler';
+import { errorHandler } from '../../prompt-helpers';
+import { t } from '../../../i18n';
+import type { IValidationLink } from '@sap-devx/yeoman-ui-types';
+import https from 'https';
+import LoggerHelper from '../../logger-helper';
+import type { AxiosRequestConfig, ODataService, ProviderConfiguration } from '@sap-ux/axios-extension';
+import { createServiceForUrl } from '@sap-ux/axios-extension';
+
+/**
+ * Structure to store validity information about url to be validated.
+ */
+interface Validity {
+    urlFormat?: boolean;
+    reachable?: boolean;
+    authRequired?: boolean;
+    authenticated?: boolean;
+    canSkipCertError?: boolean;
+}
+
+// Cert errors than may be ignored by prompt user
```

Comment:
Typo?

Reply:
Thx! Will fix is pending PR: https://github.com/SAP/open-ux-tools/pull/2036